js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/StatusDot.tsx

```diff
@@ -1,10 +1,10 @@
 import {StatusCaseDot} from './util';
-import {useAssetLiveData} from '../../asset-data/AssetLiveDataProvider';
+import {useAssetBaseData} from '../../asset-data/AssetBaseDataProvider';
 import {StatusCase, buildAssetNodeStatusContent} from '../AssetNodeStatusContent';
 import {GraphNode} from '../Utils';
 
 export function StatusDot({node}: {node: Pick<GraphNode, 'assetKey' | 'definition'>}) {
-  const {liveData} = useAssetLiveData(node.assetKey);
+  const {liveData} = useAssetBaseData(node.assetKey);
```

Comment:
I think this naming works for now (and I don't have a better idea offhand), but newcomers to the codebase might be confused that this "base data" doesn't include a lot of basic things :-p

Reply:
Oh, which basic things am I missing? Do you mean definition level things?